NuRadioReco/detector/response.py

```diff
@@ -98,6 +99,8 @@ def __init__(self, frequency, y, y_unit, time_delay=0, weight=1,
         if y[0] is None or y[1] is None:
             raise ValueError("Data for response incomplete, detected \"None\"")
 
+        # print(name, y_unit[1], y[1])
+
```

Comment:
Can we loose this please?

Reply:
Woooow. Did expect anyone to catch that. I did not want to run the ci for this but of course I can.

Reply:
I can read ... 😇

Reply:
github hack - if you put [ci skip] in your commit message it doesn't run the ci!